### ISSUE TITLE
[4.0] Atum - Backend Template - control panel heading structure Accessibility

### ISSUE THREAD

**Issue by armenos** [joomla/joomla-cms#15392]
# Description of the issues: (WCAG 2.0 Level A and Level AA) 
Back-end control panel heading structure accessibility Joomla! 4.0
There are some h5 headings before the first h1 heading in the code. Then there are some h5 headings after h1 heading. Those h5 heading after the h1 must be h2 headings.
And those h5 headings before h1 heading, I just don’t know why it’s there. 
Note: Create a logical outline of the web page with headings. Headings are about logical structure, not visual effects. Use headings in prober hierarchical order. Headings should be used to create a logical outline of the page, to allow for quick navigation to page sections.
## Possible simplified solution
Make sure that you are using heading struture to provide information about logical outline of the page, to allow for quick navigation to page sections
### How to test:
Use Mozilla Firefox extension HeadingsMap https://addons.mozilla.org/nl/firefox/addon/headingsmap/?src=search 
### Information about this issue: 
(SC 1.3.1) Info and Relationships: https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html 
(SC 1.3.2) Meaningful Sequence: https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html 
(SC 2.4.6) Headings and Labels https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html 

@C-Lodder @yvesh @ciar4n @dgt41 @wilsonge @ylahav @brianteeman

**Comment by brianteeman** [comment]
You are absolutely correct

**Comment by ciar4n** [comment]
For a frontend page I think this is easily achievable as the typical layout of any pages is generally..

Page Title
Article Title
Article SubTitle
Article Content

However in an admin backend I am not sure how reasonable of an expectation this would be. The H5 tags in question (ones before the H1) are from the private messages (top right). What you propose is possible however the layout of the admin would be very much dictated by the a11y requirement (eg. private messages moved from the header).

Would using the aria-level attribute be a suitable solution instead of changing the layout (https://www.w3.org/TR/wai-aria/states_and_properties#aria-level)?

**Comment by C-Lodder** [comment]
@armenos You're correct, it IS about logical structure. However I can't see anywhere on the web that states the `<h1>` must come first.

**Comment by C-Lodder** [comment]
I'll submit a PR for this in a bit

**Comment by C-Lodder** [comment]
https://github.com/joomla/joomla-cms/pull/15404

**Comment by armenos** [comment]
@C-Lodder H5 element from the dashboard should be h2 
H5 elements after h1
![afbeelding](https://cloud.githubusercontent.com/assets/18659157/25195969/767339d0-253f-11e7-9c34-ba77212b847d.png)
It will be beter if you call me on Skype, I’ll explain.

**Comment by C-Lodder** [comment]
PR: https://github.com/joomla/joomla-cms/pull/17048

**Comment by armenos** [comment]
Closing as PR
